src/omv/boards/ARDUINO_PORTENTA_H7/omv_boardconfig.h

```diff
@@ -131,7 +131,7 @@
 // The maximum available fb_alloc memory = FB_ALLOC_SIZE + FB_SIZE - (w*h*bpp).
 #define OMV_MAIN_MEMORY                     DTCM    // Data/BSS memory
 #define OMV_STACK_MEMORY                    DTCM    // stack memory
-#define OMV_STACK_SIZE                      (32K)
+#define OMV_STACK_SIZE                      (48K)
```

Comment:
You don't have that much stack, but there's no need anyway 32KBs is more than enough, I'd leave the rest for the future.

Reply:
Done